forge-gui/res/cardsfolder/r/river_songs_diary.txt

```diff
@@ -1,7 +1,7 @@
 Name:River Song's Diary
 ManaCost:3
 Types:Artifact
-T:Mode$ SpellCast | ValidCard$ Instant,Sorcery | Execute$ TrigEffect | TriggerZones$ Battlefield | TriggerDescription$ Imprint — Whenever a player casts an instant or sorcery spell from their hand, exile it instead of putting it into a graveyard as it resolves.
+T:Mode$ SpellCast | ValidCard$ Instant.wasCastFromHand,Sorcery.wasCastFromHand | Execute$ TrigEffect | TriggerZones$ Battlefield | TriggerDescription$ Imprint — Whenever a player casts an instant or sorcery spell from their hand, exile it instead of putting it into a graveyard as it resolves.
 SVar:TrigEffect:DB$ Effect | ReplacementEffects$ ReMoved | RememberObjects$ TriggeredCard | SubAbility$ DBImprint
 SVar:DBImprint:DB$ Pump | ImprintCards$ TriggeredCard
```

Comment:
@tool4ever the Imprint doesn't make much sense, wouldn't it be enough to use ExiledWith below?

Reply:
yes we can, I'm still planning a big cleanup for that though I wasn't sure if Imprinted should be kept for the flavor 🤷‍♂️